web/src/components/base/Link.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { helpers } from '../../constants/theme'
+import isExternalUrl from '../../utils/isExternalUrl'
+
+const StyledLink = styled(RouterLink)<{ $removeHighlight?: boolean; $adaptiveFontSize?: boolean }>`
+  color: inherit;
+  text-decoration: none;
+  display: flex;
+
+  ${props => props.$removeHighlight && helpers.removeLinkHighlighting}
+  ${props => props.$adaptiveFontSize && helpers.adaptiveFontSize}
+`
+
+const StyledAnchor = styled.a`
+  color: inherit;
+  text-decoration: none;
+`
+
+type LinkProps = {
+  to?: string
+  children: ReactNode
+  ariaLabel?: string
+  className?: string
+  newTab?: boolean
+  removeHighlight?: boolean
+  adaptiveFontSize?: boolean
+  href?: string
+  isCleanAnchor?: boolean
```

Comment:
:wrench: AFAICS this is only used once with an external url and therefore this should not be necessary as it is already handled with the check for `isExternalUrl`.